src/utils/methods.js

```diff
@@ -86,4 +86,14 @@ export const stripTags = (s) => {
 
 export const boolToStr = boolean => {
     return boolean ? 'Yes' : 'No';
-}
\ No newline at end of file
+}
+
+export const validateBadgeQR = (code, summit) => {
+    let qrCodeArray = code.split(summit.qr_registry_field_delimiter);
+
+    if (qrCodeArray.length < 2 || qrCodeArray[0] !== summit.ticket_qr_prefix) {
```

Comment:
the validate QR should handle 2 prefix
1. summit.ticket_qr_prefix for tickets scans ( originally from PurchaseOrderListPage)
2. summit.badge_qr_prefix for badge scans ( new use case)

check https://api.dev.fnopen.com/api/public/v1/summits/13 payload for instance

**order_qr_prefix: "ORDER_YOCOVIRTUALSUMMIT2021",
ticket_qr_prefix: "TICKET_YOCOVIRTUALSUMMIT2021",
badge_qr_prefix: "BADGE_YOCOVIRTUALSUMMIT2021",
qr_registry_field_delimiter: "|",**